### PR TITLE
Rule S6268: Disabling Angular built-in sanitization is security-sensitive

### LINKED ISSUE #2618: Rule S6268: Disabling Angular built-in sanitization is security-sensitive
We should detect following functions from Angular's `DomSanitizer` object as security hotspot [S6268](https://jira.sonarsource.com/browse/RSPEC-6268)

```
DomSanitizer.bypassSecurityTrustHtml(value: string)
DomSanitizer.bypassSecurityTrustStyle(value: string)
DomSanitizer.bypassSecurityTrustScript(value: string)
DomSanitizer.bypassSecurityTrustUrl(value: string)
DomSanitizer.bypassSecurityTrustResourceUrl(value: string)
```

Related rule S5247 is about disabling escaping in the **whole** application, so this is another rule.

### PR DESCRIPTION
**Pull request by vilchik-elena**
Fixes #2618

### CODE REVIEW

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S6268.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<p>Still, developers have the ability to manually mark a value as trusted if they are sure that the value is already sanitized. Accidentally trusting
+malicious data will introduce an XSS vulnerability in the application and enable a wide range of serious attacks like accessing/modifying sensitive
+information or impersonating other users.</p>
+<h2>Ask Yourself Whether</h2>
+<ul>
+  <li> The value for which sanitization has been disabled is user controlled. </li>
+  <li> It's difficult to understand how this value is constructed. </li>
+</ul>
+<p>There is a risk if you answered yes to any of those questions.</p>
+<h2>Recommended Secure Coding Practices</h2>
```
Comment (vilchik-elena) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S6268.html:12]: code samples are missing (ping @pierre-loup-tristant-sonarsource )

File: eslint-bridge/yarn.lock [hunk 2]
```diff
@@ -1846,10 +1846,10 @@ eslint-plugin-chai-friendly@0.6.0:
   resolved "https://registry.yarnpkg.com/eslint-plugin-chai-friendly/-/eslint-plugin-chai-friendly-0.6.0.tgz#54052fab79302ed0cea76ab997351ea4809bfb77"
   integrity sha512-Uvvv1gkbRGp/qfN15B0kQyQWg+oFA8buDSqrwmW3egNSk/FpqH2MjQqKOuKwmEL6w4QIQrIjDp+gg6kGGmD3oQ==
 
-eslint-plugin-sonarjs@0.7.0:
-  version "0.7.0"
-  resolved "https://registry.yarnpkg.com/eslint-plugin-sonarjs/-/eslint-plugin-sonarjs-0.7.0.tgz#2dbad45e03a6a4de687756b659e88bf3f977383f"
-  integrity sha512-vi6zGkd5Eznc32AQlleWUOMrMeDiUih9wR7nPPfrDCyVRmwYNHIBRPZGv1EgXwELwaPghCSvoAoHoR7uSbBF/Q==
+eslint-plugin-sonarjs@0.7.1-102:
```
Comment (vilchik-elena) [inline eslint-bridge/yarn.lock:1849]: this change should be reverted probably

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S6268.html (hunk 1 above)
Comment (pierre-loup-tristant-sonarsource) [reply javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S6268.html:12]: @vilchik-elena code example have been added to the RSPEC

**Comment by pierre-loup-tristant-sonarsource** [comment]
@vilchik-elena I did minor changes in [RSPEC-6268](https://jira.sonarsource.com/browse/RSPEC-6268). Rule metadata from this PR are no more up to date.

**Review by saberduck — APPROVED** [review]
lgtm!

**Pull request merged**